libfwupd/fwupd-common.c

```diff
@@ -152,6 +152,15 @@ fwupd_get_os_release (GError **error)
 	g_autofree gchar *buf = NULL;
 	g_auto(GStrv) lines = NULL;
 
+/* TODO: Read the Windows version */
```

Comment:
Do you know how we can we do that?

Reply:
Looks like this: https://docs.microsoft.com/en-us/windows/win32/sysinfo/getting-the-system-version

Reply:
I think my point was more that we should probably do the extra bits like this before merging the branch.